Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -86,11 +86,20 @@ public extension UIView {
 
 	/// SwifterSwift: First responder.
 	public var firstResponder: UIView? {
```

Comment:
We may want to consider making this a function now and mention it's recursive behavior in the comment.
`/// SwifterSwift: Recursively find the first responder.`